src/Dotnet.Script/Program.cs

```diff
@@ -98,7 +98,7 @@ private static int Wain(string[] args)
                     if (!string.IsNullOrWhiteSpace(code.Value))
                     {
                         var optimizationLevel = OptimizationLevel.Debug;
-                        if (configuration.HasValue() && configuration.Value().ToLower() == "release")
+                        if (configuration.ValueEquals("release", StringComparison.OrdinalIgnoreCase))
```

Comment:
`StringComparison.OrdinalIgnoreCase` could be the default here. That would clean up the call site ever more

Reply:
That can be done post-PR as you see fit? Meanwhile, I didn't want to burn a global policy into an extension method.

Reply:
I was thinking setting it as a default value for the parameter. That way it is not "burned" into the method :)

Reply:
Took a slightly different approach in 8fd017d3cb2a45aae5bfc904f7ae970cf736a25c, which kills two birds with one stone (the second bird is the `"release"` literal).